pygorithm/sorting/cocktail_sort.py

```diff
@@ -0,0 +1,57 @@
+'''
+Created by: Pratik Narola (https://github.com/Pratiknarola)
+last modified: 14-10-2019
+'''
+
+'''
+
+Cocktail Sort is a variation of Bubble sort. 
+The Bubble sort algorithm always traverses elements from left
+and moves the largest element to its correct position in first iteration 
+and second largest in second iteration and so on. 
+Cocktail Sort traverses through a given array in both directions alternatively.
+
+'''
+
+def cocktailSort(a): 
+    n = len(a) 
+    swapped = True
+    start = 0
+    end = n-1
+    while (swapped == True): 
+  
+        # reset the swapped flag on entering the loop, 
+        # because it might be true from a previous 
+        # iteration. 
+        swapped = False
+  
+        # loop from left to right same as the bubble 
+        # sort 
+        for i in range (start, end): 
```

Comment:
Can remove the space after "range": https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements